### PR TITLE
pd: add more region health info

### PR DESCRIPTION
**Pull request by lhy1024**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: daily-check.md [hunk 1]
```diff
@@ -44,10 +44,13 @@ TiDB 作为分布式数据库，对比单机数据库机制更加复杂，其自
 
 + `miss-peer-region-count`：缺副本的 Region 数量，不会一直大于 0。
 + `extra-peer-region-count`：多副本的 Region 数量，调度过程中会产生。
-+ `empty-region-count`：空 Region 的数量，一般是 `TRUNCATE TABLE`/`DROP TABLE` 语句导致。如果数量较多，可以考虑开启跨表 Region merge。
++ `empty-region-count`：空 Region 的数量，大小小于 1MB。一般是 `TRUNCATE TABLE`/`DROP TABLE` 语句导致。如果数量较多，可以考虑开启跨表 Region merge。
```
Comment (lilin90) [inline daily-check.md:47]:
```suggestion
+ `empty-region-count`：空 Region 的数量，大小小于 1 MiB。一般是 `TRUNCATE TABLE`/`DROP TABLE` 语句导致。如果数量较多，可以考虑开启跨表 Region merge。
```

File: daily-check.md [hunk 2]
```diff
@@ -44,10 +44,13 @@ TiDB 作为分布式数据库，对比单机数据库机制更加复杂，其自
...
 + `pending-peer-region-count`：Raft log 落后的 Region 数量。由于调度产生少量的 pending peer 是正常的，但是如果 pending peer 的数量持续（超过 30 分钟）很高，可能存在问题。
 + `down-peer-region-count`：Raft leader 上报有不响应 peer 的 Region 数量。
 + `offline-peer-region-count`：peer 下线过程中的 Region 数量。
++ `learner-peer-region-count`：learner peer 的数量。来源可能是多个，比如 TiFlash 上面是 learner peer，比如配置的 Placement Rule 如果包含 learner peer 也会有 learner peer。
```
Comment (lilin90) [inline daily-check.md:51]:
```suggestion
+ `learner-peer-region-count`：learner peer 的数量。来源可能是多个，例如 TiFlash 上的 learner peer，以及配置的 Placement Rules 包含 learner peer。
```

File: daily-check.md [hunk 3]
```diff
@@ -44,10 +44,13 @@ TiDB 作为分布式数据库，对比单机数据库机制更加复杂，其自
...
++ `oversized-region-count`: Region 大小大于 `region-max-size` 或 `region-max-keys` 的 Region 数量。
++ `undersized-region-count`: Region 大小小于 `max-merge-region-size` 或 `max-merge-region-keys` 的 Region 数量。
```
Comment (lilin90) [inline daily-check.md:53]:
```suggestion
+ `oversized-region-count`：Region 大小大于 `region-max-size` 或 `region-max-keys` 的 Region 数量。
+ `undersized-region-count`：Region 大小小于 `max-merge-region-size` 或 `max-merge-region-keys` 的 Region 数量。
```

**Review by lilin90 — COMMENTED** [review]
Line 41 放的截图是不是也过时了需要更新下？
`Region 信息面板`

**Review by nolouch — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

File: daily-check.md [hunk 4]
```diff
@@ -42,12 +42,15 @@ TiDB 作为分布式数据库，对比单机数据库机制更加复杂，其自
...
 以上 Region 信息面板说明如下：
 
-+ `miss-peer-region-count`：缺副本的 Region 数量，不会一直大于 0。
-+ `extra-peer-region-count`：多副本的 Region 数量，调度过程中会产生。
-+ `empty-region-count`：空 Region 的数量，一般是 `TRUNCATE TABLE`/`DROP TABLE` 语句导致。如果数量较多，可以考虑开启跨表 Region merge。
-+ `pending-peer-region-count`：Raft log 落后的 Region 数量。由于调度产生少量的 pending peer 是正常的，但是如果 pending peer 的数量持续（超过 30 分钟）很高，可能存在问题。
 + `down-peer-region-count`：Raft leader 上报有不响应 peer 的 Region 数量。
++ `empty-region-count`：空 Region 的数量，大小小于 1 MiB。一般是 `TRUNCATE TABLE`/`DROP TABLE` 语句导致。如果数量较多，可以考虑开启跨表 Region merge。
++ `extra-peer-region-count`：多副本的 Region 数量，调度过程中会产生。
++ `learner-peer-region-count`：learner peer 的数量。来源可能是多个，例如 TiFlash 上的 learner peer，以及配置的 Placement Rules 包含 learner peer。
```
Comment (lhy1024) [inline daily-check.md:48]:
```suggestion
+ `learner-peer-region-count`：含有 learner peer 的 Region 数量。来源可能是多个，例如 TiFlash 上的 learner peer，以及配置的 Placement Rules 包含 learner peer。
```

File: daily-check.md [hunk 5]
```diff
@@ -42,12 +42,15 @@ TiDB 作为分布式数据库，对比单机数据库机制更加复杂，其自
...
 以上 Region 信息面板说明如下：
 
-+ `miss-peer-region-count`：缺副本的 Region 数量，不会一直大于 0。
-+ `extra-peer-region-count`：多副本的 Region 数量，调度过程中会产生。
-+ `empty-region-count`：空 Region 的数量，一般是 `TRUNCATE TABLE`/`DROP TABLE` 语句导致。如果数量较多，可以考虑开启跨表 Region merge。
-+ `pending-peer-region-count`：Raft log 落后的 Region 数量。由于调度产生少量的 pending peer 是正常的，但是如果 pending peer 的数量持续（超过 30 分钟）很高，可能存在问题。
 + `down-peer-region-count`：Raft leader 上报有不响应 peer 的 Region 数量。
++ `empty-region-count`：空 Region 的数量，大小小于 1 MiB。一般是 `TRUNCATE TABLE`/`DROP TABLE` 语句导致。如果数量较多，可以考虑开启跨表 Region merge。
++ `extra-peer-region-count`：多副本的 Region 数量，调度过程中会产生。
++ `learner-peer-region-count`：含有 learner peer 的 Region 数量。来源可能是多个，例如 TiFlash 上的 learner peer，以及配置的 Placement Rules 包含 learner peer。
```
Comment (lilin90) [inline daily-check.md:48]:
```suggestion
+ `learner-peer-region-count`：含有 learner peer 的 Region 数量。learner peer 的来源可能是多个，例如 TiFlash 上的 learner peer，以及配置的 Placement Rules 包含 learner peer。
```

**Review by hfxsd — APPROVED** [review]

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**